js/hangman.js

```diff
@@ -0,0 +1,129 @@
+let pass = "Bez pracy nie ma kołaczy";
+pass = pass.toUpperCase();
+
+let passHidden = "";
+
+let passLen = pass.length;
+
+let missedCount = 0;
+
+let yes = new Audio("samples/yes.wav");
+let no = new Audio("samples/no.wav");
+
+for (i=0; i<passLen; i++)
+{
+    if (pass.charAt(i) == " ") passHidden = passHidden + " ";
+    else passHidden = passHidden + "-";
+}
+
+function writePass()
+{
+    document.getElementById("label").innerHTML = passHidden;
+}
+
+window.onload = start;
+
+let letters = new Array(35);
+
+letters[0] = "A";
+letters[1] = "Ą";
+letters[2] = "B";
+letters[3] = "C";
+letters[4] = "Ć";
+letters[5] = "D";
+letters[6] = "E";
+letters[7] = "Ę";
+letters[8] = "F";
+letters[9] = "G";
+letters[10] = "H";
+letters[11] = "I";
+letters[12] = "J";
+letters[13] = "K";
+letters[14] = "L";
+letters[15] = "Ł";
+letters[16] = "M";
+letters[17] = "N";
+letters[18] = "Ń";
+letters[19] = "O";
+letters[20] = "Ó";
+letters[21] = "P";
+letters[22] = "Q";
+letters[23] = "R";
+letters[24] = "S";
+letters[25] = "Ś";
+letters[26] = "T";
+letters[27] = "U";
+letters[28] = "V";
+letters[29] = "W";
+letters[30] = "X";
+letters[31] = "Y";
+letters[32] = "Z";
+letters[33] = "Ż";
+letters[34] = "Ź";
+
+function start()
+{
+    let divContent = "";
+
+    for (i=0; i<35; i++)
```

Comment:
Try to avoid using magic numbers if its possible. You can use here `length` property from Array prototype instead of using magic number.